app/models/user.rb

```diff
@@ -11,6 +11,10 @@ class User < ApplicationRecord
     :rememberable, :trackable, :timeoutable, :validatable, :lockable
 
   validates :name, presence: true, on: :update
+  validates :password, presence: true,
+    confirmation: true,
```

Comment:
Should this line be here? Would have expected it to fail some tests if not, but surprises me.

Reply:
I would have thought removing the `confirmation: true` is what we want. I do recall this being a bit buggy when implementing it. But now we don't have confirmations we don't need it at all